### DISCUSSION TITLE
jspdf-autotable library problem

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#10110] [Q&A]
I'm currently trying to export tables as PDF files, so I've introduced the library "jspdf-autotable". There's no problem when running it locally, but after packaging it into Docker and attempting to export, an error occurs: "TypeError: MA.autoTable is not a function". I'm wondering what's causing this problem. Is it because the way of importing is incorrect? Could someone teach me about this? I can't find this package in the node_modules either.

**Comment by o1lab** [comment]
Hard to know as its customised docker container. Please put your findings here for any future discussions.
